envo.owl

```diff
@@ -143489,6 +143489,696 @@ Classes for population already  exist in IDO (&apos;organism population&apos;, I
     
 
 
+    <!-- http://purl.obolibrary.org/obo/envo.owl#OWLClass_03510000 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/envo.owl#OWLClass_03510000">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/ENVO_00002008"/>
+        <obo:IAO_0000115>A light- to dark-colored granular solid mixture that is generated when timber is processed, such as when it is chipped, sawed, turned, drilled or sanded. The chemical composition of wood dust depends on the species of tree and consists mainly of cellulose, polyoses and lignin, with a large and variable number of substances with lower relative molecular mass. Wood dust is not typically produced for specific uses. However, it is used commercially for wood composts. The primary routes of potential human exposure to wood dust are inhalation and dermal contact. It is known to be a human carcinogen.</obo:IAO_0000115>
```

Comment:
Please format definitions close to the "An A is a B which Cs" model. It's better if these are quite short and focus on the essence of things. Details can be moved to an rdfs:comment. 

For example, this definition:
> A light- to dark-colored granular solid mixture that is generated when timber is processed, such as when it is chipped, sawed, turned, drilled or sanded. The chemical composition of wood dust depends on the species of tree and consists mainly of cellulose, polyoses and lignin, with a large and variable number of substances with lower relative molecular mass. Wood dust is not typically produced for specific uses. However, it is used commercially for wood composts. The primary routes of potential human exposure to wood dust are inhalation and dermal contact. It is known to be a human carcinogen.

Would be converted into:


>Definition: Dust which is 1) generated when wood is processed and 2) composed primarily of cellulose, polyoses, and lignin, with smaller quantities of a large and diverse range of substances with relatively low molecular mass.


>Comment: Wood dust is typically generated through processes where wood is chipped, sawed, turned, drilled or sanded. The chemical composition of wood dust depends on the species of tree. Wood dust is not typically produced for specific uses. However, it is used commercially for wood composts. The primary routes of potential human exposure to wood dust are inhalation and dermal contact. It is known to be a human carcinogen.


note that I changed "timber" to "wood" to keep things more consistent and avoid issues where this dust may not be generated from what people would consider "timber". You could create a subclass for timber dust if you think it's needed, or add it as a narrow synonym.

Reply:
Good recommendations, thanks @pbuttigieg. You can read some of my thoughts on definitions here, couched informally, but I think these should be turned into formal OBO recommendations: https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/

Reply:
addressed